modules/ossm-tutorial-bookinfo-verify-install.adoc

```diff
@@ -47,3 +46,33 @@ reviews-v3-6dfd49b55b-vcwpf       2/2     Running   0          12m
 echo "http://$GATEWAY_URL/productpage"
 ----
 . Copy and paste the output in a web browser to verify the Bookinfo product page is deployed.
+
+.Procedure from Kiali web console
+
+. Obtain the address for the Kiali web console.
+
+.. Log in to the {product-title} web console as a user with cluster-admin rights. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+.. Navigate to *Networking* -> *Routes*.
+
+.. On the *Routes* page, select the control plane project, for example `istio-system`, from the *Namespace* menu.
++
+The *Location* column displays the linked address for each route.
++
+
+.. Click the link in the *Location* column for Kiali.
+
+.. Click *Log In With OpenShift*. The Kiali *Overview* screen presents tiles for each project namespace.
+
+. In Kiali, click *Graph*.
+
+. Select bookinfo from the *Namespace* list, then select App graph.
++
+image::ossm-kiali-graph-bookinfo.png[Kiali displaying bookinfo application]
++
+If no graph appears in the console, then the application might not have obtained data. To update the data:
++
+* If the empty graph message appears, then click *Display idle nodes*
```

Comment:
```suggestion
* If the empty graph message appears, then enable *Display idle nodes* from the *Display* menu.
  * This display nodes that are dfined but have never received or sent requests.  It can confirm that your application is properly defined but that no request traffic has been reported.
```

Reply:
Made this edit